Assets/MRTK/Core/Providers/UnityInput/UnityJoystickManager.cs

```diff
@@ -100,130 +101,124 @@ public override void Disable()
             ActiveControllers.Clear();
         }
 
+        private static readonly ProfilerMarker GetActiveControllersPerfMarker = new ProfilerMarker("[MRTK] UnityJoystickManager.GetActiveControllers");
+
         /// <inheritdoc/>
         public override IMixedRealityController[] GetActiveControllers()
         {
-            Profiler.BeginSample("[MRTK] UnityJoystickManager.GetActiveControllers");
-
-            IMixedRealityController[] controllers = ActiveControllers.Values.ToArray<IMixedRealityController>();
-
-            Profiler.EndSample(); // GetActiveControllers
-
-            return controllers;
+            using (GetActiveControllersPerfMarker.Auto())
+            {
+                IMixedRealityController[] controllers = ActiveControllers.Values.ToArray<IMixedRealityController>();
+                return controllers;
+            }
         }
+        private static readonly ProfilerMarker RefreshDevicesPerfMarker = new ProfilerMarker("[MRTK] UnityJoystickManager.Update");
```

Comment:
nit, to match the others
```suggestion

        private static readonly ProfilerMarker RefreshDevicesPerfMarker = new ProfilerMarker("[MRTK] UnityJoystickManager.Update");
```

Reply:
There's another mistake in this marker... The string has the wrong method name.